main.cpp

```diff
@@ -227,39 +249,42 @@ struct Compressor
         void oversample(bool oversampling);
     };
 
-    //compress audio
     void compress(double threshold, float ratio, int compType);
-    //make-up volume gain
     void volumeMakeUp(float inputGain, float outputGain);
-    //saturate input
     void saturateInput(Saturator sat);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise